pkg/operator/endpoints/logs.go

```diff
@@ -67,3 +70,57 @@ func ReadLogs(w http.ResponseWriter, r *http.Request) {
 	}
 	operator.StreamLogsFromRandomPod(labels, socket)
 }
+
+func LogURL(w http.ResponseWriter, r *http.Request) {
+	apiName := mux.Vars(r)["apiName"]
+	jobID := getOptionalQParam("jobID", r)
+
+	if jobID != "" {
+		JobLogURL(w, r)
+		return
+	}
+
+	deployedResource, err := resources.GetDeployedResourceByName(apiName)
+	if err != nil {
+		respondError(w, r, err)
+		return
+	}
+
+	if deployedResource.Kind == userconfig.BatchAPIKind || deployedResource.Kind == userconfig.TaskAPIKind {
+		respondError(w, r, ErrorLogsJobIDRequired(*deployedResource))
+		return
+	}
+
+	switch deployedResource.Kind {
+	case userconfig.AsyncAPIKind:
+		apiResponse, err := asyncapi.GetAPIByName(deployedResource)
+		if err != nil {
+			respondError(w, r, err)
+			return
+		}
+		logURL, err := operator.APILogURL(apiResponse[0].Spec)
+		if err != nil {
+			respondError(w, r, err)
+			return
+		}
+		respondJSON(w, r, schema.LogResponse{
+			LogURL: logURL,
+		})
+	case userconfig.RealtimeAPIKind:
+		apiResponse, err := realtimeapi.GetAPIByName(deployedResource)
+		if err != nil {
+			respondError(w, r, err)
+			return
+		}
+		logURL, err := operator.APILogURL(apiResponse[0].Spec)
+		if err != nil {
+			respondError(w, r, err)
+			return
+		}
+		respondJSON(w, r, schema.LogResponse{
+			LogURL: logURL,
+		})
+	default:
+		respondError(w, r, resources.ErrorOperationIsOnlySupportedForKind(*deployedResource, userconfig.RealtimeAPIKind))
```

Comment:
Should this be `resources.ErrorOperationIsOnlySupportedForKind(*deployedResource, userconfig.RealtimeAPIKind, userconfig.AsyncAPIKind)`?